src/main/java/org/cryptomator/ui/health/HealthCheckModule.java

```diff
@@ -84,12 +86,15 @@ static FxmlLoaderFactory provideFxmlLoaderFactory(Map<Class<? extends FxControll
 	@Provides
 	@HealthCheckWindow
 	@HealthCheckScoped
-	static Stage provideStage(StageFactory factory, @MainWindow Stage owner, ResourceBundle resourceBundle, ChangeListener<Boolean> showingListener) {
+	static Stage provideStage(@Nullable Stage windowToClose, StageFactory factory, @MainWindow Stage owner, ResourceBundle resourceBundle, ChangeListener<Boolean> showingListener) {
+		if (windowToClose != null) {
+			windowToClose.close();
```

Comment:
This is a side effect. Don't close it in a `@Provides` method, but rather within `HealthCheckComponent.showHealthCheckWindow()` or during initialization of the `StartController`

Reply:
Fixed in 03a7e89